source/heating_model/interface.cc

```diff
@@ -228,6 +228,13 @@ namespace aspect
                             const MaterialModel::MaterialModelOutputs<dim> &material_model_outputs,
                             HeatingModel::HeatingModelOutputs &heating_model_outputs) const
     {
+      // Initialize the outputs to zero
+      for (unsigned int q=0; q<heating_model_outputs.heating_source_terms.size(); ++q)
+        {
+          heating_model_outputs.heating_source_terms[q] = 0.0;
+          heating_model_outputs.lhs_latent_heat_terms[q] = 0.0;
+        }
+
       // the heating outputs are initialized with zeros, so there is no heating if they are not set
```

Comment:
And maybe you could change this to "individual heating outputs", so it is clear what the difference is (compared to the comment above).